agent/exec/container/adapter.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/docker/swarmkit/log"
 	"github.com/docker/swarmkit/protobuf/ptypes"
 	"golang.org/x/net/context"
+	"golang.org/x/time/rate"
```

Comment:
You need to vendor this package. Unfortunately godep is very painful to work with. Let me know if you need help.